### ISSUE TITLE
OpenLiteSpeed 1.6.4 Ubuntu 18.04 Build Error.

### ISSUE THREAD

**Issue by KotoishiHeart** [litespeedtech/openlitespeed#152]
~~~
git clone -v1_6 https://github.com/litespeedtech/openlitespeed/
cd openlitespeed/
sudo ./build.sh
~~~

result
~~~
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libexpat.a(xmlrole.o): relocation R_X86_64_32S against `.text' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libIP2Location.a(IP2Location.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libIP2Location.a(IP2Loc_DBInterface.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libudns.a(udns_resolver.o): relocation R_X86_64_32 against undefined symbol `dns_defctx' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libudns.a(udns_init.o): relocation R_X86_64_32 against undefined symbol `dns_defctx' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libudns.a(udns_rr_a.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libudns.a(udns_rr_ptr.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libudns.a(udns_dn.o): relocation R_X86_64_32 against symbol `dns_inaddr_arpa_dn' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: /home/ubuntu/openlitespeed-1.6.4/../thirdparty/lib/libudns.a(udns_parse.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status
src/CMakeFiles/openlitespeed.dir/build.make:137: recipe for target 'src/openlitespeed' failed
make[2]: *** [src/openlitespeed] Error 1
CMakeFiles/Makefile2:211: recipe for target 'src/CMakeFiles/openlitespeed.dir/all' failed
make[1]: *** [src/CMakeFiles/openlitespeed.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
cp: cannot stat 'src/openlitespeed': No such file or directory
cp: cannot stat 'src/modules/modreqparser/*.so': No such file or directory
cp: cannot stat 'src/modules/cache/*.so': No such file or directory
cp: cannot stat 'src/modules/modinspector/*.so': No such file or directory
cp: cannot stat 'src/modules/uploadprogress/*.so': No such file or directory
Start to pack files.
-e Building finished, please run ./install.sh for installation.
-e You may want to update the ols.conf to change the settings before installation.
-e Enjoy.
~~~

How can I build?

**Comment by litespeedtech** [comment]
I will do test on that ubuntu 18 soon.
By the way, you may want to use our pre-built package directly if you want.
The url is 
https://openlitespeed.org/packages/openlitespeed-1.6.4.tgz
Thanks you for you feedback. 
David

**Comment by litespeedtech** [comment]
I tested on Ubuntu 18.04 and I do found error.
It is fixed and we will release 1.6.5 soon.
Thank you
David

**Comment by litespeedtech** [comment]
It is fixed in 1.6.5 which was just released.
